src/controller/python/test/test_scripts/network_commissioning.py

```diff
@@ -198,7 +198,7 @@ async def test_wifi(self, endpointId):
         logger.info(f"Connect to a network")
         req = Clusters.NetworkCommissioning.Commands.ConnectNetwork(
             networkID=TEST_WIFI_SSID.encode(), breadcrumb=self.with_breadcrumb())
-        res = await self._devCtrl.SendCommand(nodeid=self._nodeid, endpoint=endpointId, payload=req)
+        res = await self._devCtrl.SendCommand(nodeid=self._nodeid, endpoint=endpointId, payload=req, interactionTimeoutMs=120000)
```

Comment:
Where is this number derived from?

Most platforms expose a value of around 20 - 30s for the `NetworkCommissioningCluster.ConnectMaxTimeSeconds` attribute (which is what is used to determine the timeout for this operation in the auto commissioner). 

I'd recommend we set this to max there (i.e 30s).

Reply:
I mistook `interactionTimeoutMs` as the 'expected time spent processing/handling the command on the target' when instead, it is the total absolute timeout for the interaction.

Please use `devCtrl.ComputeRoundTripTimeout` to actually compute this value correctly. You can pass in 30s to that function to do so.